src/i18n/locale/en.json

```diff
@@ -231,48 +231,48 @@
       "browse": "Browse",
       "display": "Display",
       "filter": "Filter",
-      "loading": "Loading...",
+      "loading": "Loading…",
       "sort": "Sort"
     },
     "language": {
       "label": {
         "language": "Language",
-        "language_with_code": "language with code: {{code}}"
+        "language_with_code": "Language with code: {{code}}"
       },
       "title": {
-        "enabled_languages": "Enabled Languages"
+        "enabled_languages": "Used Languages"
       }
     },
     "sort": {
       "label": {
-        "by_fetch_date": "By Fetch date",
-        "by_source": "By Source"
+        "by_fetch_date": "By date fetched",
+        "by_source": "By source"
       }
     }
   },
   "library": {
     "error": {
       "label": {
-        "empty": "Your Library is empty",
-        "no_matches": "There are no Manga matching this filter"
+        "empty": "Your library is empty",
+        "no_matches": "No manga matches this filter"
       }
     },
     "option": {
       "display": {
         "badge": {
           "label": {
-            "download_badges": "Download Badges",
-            "unread_badges": "Unread Badges"
+            "download_badges": "Download badges",
+            "unread_badges": "Unread badges"
           },
           "title": "Badges"
         }
       },
       "sort": {
         "label": {
-          "alphabetically": "Alphabetically",
-          "by_date_added": "By Date Added",
-          "by_last_read": "By Last Read",
-          "by_unread_chapters": "By Unread chapters"
+          "alphabetically": "A-Z",
+          "by_date_added": "Most recently added",
+          "by_last_read": "Most recently read",
+          "by_unread_chapters": "Number of unread chapters"
```

Comment:
I'm not comfortable with these changes, can you explain the reasoning?
cc @schroda

Reply:
Fewer abstractions. The date relates to time, as it is either oldest first, or implied to be the other way around.

Reply:
I prefer the way it's done in tachiyomi (without "By ..." or "Most recently ..."
![Screenshot 2023-03-31 183432](https://user-images.githubusercontent.com/50052685/229178692-32893473-c605-4008-9590-5096ea778c1b.png)